data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/rpc/queries/ByIdsRpcQuery.avsc

```diff
@@ -2,15 +2,15 @@
   "type": "record",
   "name": "ByIdsRpcQuery",
   "namespace": "net.corda.data.crypto.wire.ops.rpc.queries",
-  "doc": "Request to lookup for keys by their ids, the maximum number of items is 20.",
+  "doc": "Request to lookup for keys by their ids. Ids can be either short or full key ids. The maximum number of items is 20",
   "fields": [
     {
-      "name": "keys",
-      "type": {
-        "type": "array",
-        "items": "string"
-      },
-      "doc": "List of key ids."
+      "name": "keyIds",
+      "type": [
+        "net.corda.data.crypto.ShortHashes",
+        "net.corda.data.crypto.SecureHashes"
+        ],
```

Comment:
whitespace looks wrong - the closing ] should be algiedn with the open double quote of "type"

Reply:
Addressed